src/ls.js

```diff
@@ -162,6 +166,18 @@ function _ls(options, paths) {
   list.toEnd = function(file) {
     common.wrap('toEnd', _toEnd, {idx: 1}).call(this.stdout, file);
   };
+  list.cat = function() {
+    return common.wrap('cat', _cat, {idx: 1}).apply(this.stdout, arguments);
```

Comment:
I actually just started refactoring `ShellString()` to have a similar affect. It still takes 2 arguments, but the first is the stdout and the second is the stderr. stdout can be either an array or a string, and it'll return either a "ShellArray" (an array with all the extra methods), or a `ShellString`, respectively. This simplifies the code in `src/ls.js`, which I suspect is what you were going for.

I've also fixed the re-wrapping issue, and removed all the extra `require()`s from `src/ls.js`.